server/utils/routes.js

```diff
@@ -3,7 +3,7 @@ const userRoot = `${v1Root}/users/:uid`
 
 export const routes = {
   root: `/`,
-  chart: '/charts/:chart_id',
+  chart: '/charts/data/:chart_id',
```

Comment:
why does this route need to change?

Reply:
we already have a charts/:chart_id route that handles edit, delete, and getting the form. This route would get the data needed to render the chart

Reply:
oh I see. let's try a nested route then like `chartData: '/charts/:chart_id/data'`.